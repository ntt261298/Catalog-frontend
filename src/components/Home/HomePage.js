import React from 'react';
import 'style/home.css';
import Category from 'components/Commons/Category';
import Item from 'components/Commons/Item';
import Header from 'components/Layout/Header';

export const HomePage = () => (
  <div>
    <Header />
    <div className="home-main">
      <Category />
      <Item type="home" id="" />
    </div>
  </div>
);

export default HomePage;
