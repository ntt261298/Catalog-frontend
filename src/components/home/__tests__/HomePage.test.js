import React from 'react';
import { shallow } from '../../../enzyme';
import { HomePage } from '../HomePage';

describe('src/components/home/HomePage.js', () => {
  let wrapper;

  const setup = () => {
    wrapper = shallow(
      <HomePage />,
    );
  };

  it('should render home page', () => {
    setup();
    // Expect the wrapper object to be defined
    expect(wrapper).toMatchSnapshot();
  });
});
