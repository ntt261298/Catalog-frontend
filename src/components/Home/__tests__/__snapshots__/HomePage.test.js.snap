// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home/HomePage should render Home page 1`] = `
<div>
  <Connect(Component) />
  <div
    className="home-main"
  >
    <Connect(Category) />
    <Connect(Item)
      id=""
      type="home"
    />
  </div>
</div>
`;
